x-pack/plugins/fleet/public/applications/integrations/app.tsx

```diff
@@ -73,17 +77,20 @@ export const IntegrationsAppContext: React.FC<{
                       <UIExtensionsContext.Provider value={extensions}>
                         <FleetStatusProvider>
                           <startServices.customIntegrations.ContextProvider>
-                            <Router history={history}>
-                              <AgentPolicyContextProvider>
-                                <PackageInstallProvider
-                                  notifications={startServices.notifications}
-                                  theme$={theme$}
-                                >
-                                  <IntegrationsHeader {...{ setHeaderActionMenu, theme$ }} />
-                                  {children}
-                                </PackageInstallProvider>
-                              </AgentPolicyContextProvider>
-                            </Router>
+                            <CloudContext>
+                              <Router history={history}>
+                                <AgentPolicyContextProvider>
+                                  <PackageInstallProvider
+                                    notifications={startServices.notifications}
+                                    theme$={theme$}
+                                  >
+                                    <IntegrationsHeader {...{ setHeaderActionMenu, theme$ }} />
+                                    {children}
+                                    <Chat />
```

Comment:
To continue the discussion started in https://github.com/elastic/kibana/pull/123339#discussion_r789408415: 

The points you listed in your reply are very valid, so I'm fine keeping the context and the components separated. The notable difference is that with the current approach, all potential consumers of the chat feature will have to define cloud as a required bundle in addition to being an optional plugin dependency, but I don't think this really have any impact given that we're not disabling the cloud plugin on non-cloud environments, so the cloud bundle is loaded anyway.